HomeWork/src/main/java/string/StringLevelThree.java

```diff
@@ -0,0 +1,52 @@
+package string;
+
+import java.util.Arrays;
+
+public class StringLevelThree {
+
+    public int countYZ (String str) {
+        int counter = 0;
+        str = str.toLowerCase();
+
+        for (int i = 0; i < str.length(); i++) {
+            if (str.charAt(i) == 'z' || str.charAt(i) == 'y') {
+                if (i == str.length() -1) {
+                    counter++;
+                    break;
+                } else if (Character.isLetter(str.charAt(i + 1))) {
+                   continue;
+                } else {
+                    counter++;
+                }
+            }
+        }
+        return counter;
+    }
+
+    public String withoutString (String base, String remove) {
+        String str = "";
```

Comment:
Yep, it seems this solution is not working. But anyway. You did a great job, you've solved so many tasks already. I would suggest you keep going on `codingbat` tasks and you will definitely solve this one and many more.